packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -459,41 +454,18 @@ export function isNetwork(chainId: ChainId) {
   const isXaiTestnet = chainId === ChainId.XaiTestnet
   const isStylusTestnet = chainId === ChainId.StylusTestnet
 
-  const ethereumChainIds = [
-    ChainId.Mainnet,
-    ChainId.Rinkeby,
-    ChainId.Goerli,
-    ChainId.Sepolia,
-    ChainId.Local
-  ]
-
-  const isEthereum = ethereumChainIds.includes(chainId)
-
-  const customArbitrumChainIds = customChains
-    .filter(chain => ethereumChainIds.includes(chain.partnerChainID))
-    .map(chain => chain.chainID)
-
-  const arbitrumChainIds = [
-    ChainId.ArbitrumOne,
-    ChainId.ArbitrumNova,
-    ChainId.ArbitrumGoerli,
-    ChainId.ArbitrumRinkeby,
-    ChainId.ArbitrumLocal,
-    ChainId.ArbitrumSepolia
-  ]
+  const isEthereum = isMainnet || isRinkeby || isGoerli || isSepolia || isLocal
 
   const isArbitrum =
     isArbitrumOne ||
     isArbitrumNova ||
     isArbitrumGoerli ||
     isArbitrumRinkeby ||
     isArbitrumLocal ||
-    isArbitrumSepolia ||
-    customArbitrumChainIds.includes(chainId)
+    isArbitrumSepolia
```

Comment:
This won't work when there are mainnet Orbit chains?

Reply:
This should work, it was just part of confusion about what custom chains are meant to be (whether just custom Orbit chains or possibly L2 as well). But there will be no custom L2 added therefore this check is not needed.